tools/pot/openvino/tools/pot/graph/passes.py

```diff
@@ -876,11 +877,17 @@ def insert_fake_quantize(graph, node, ports=None, names=None, fq_types=None, hw_
         if hw_config is not None and hw_config[node_type]:
             fq_configs = hw_config[node_type][fq_group]
 
+        if node_type in input_priority_types:
```

Comment:
@AlexKoff88, what do you think about this priority types feature? We should change it in the future somehow because this is a little bit inconvenient to develop, IMHO.

Reply:
Can I use this priority types feature? or I should find another feature to implement?

Reply:
Can anybody explain what kind of problem we are trying to solve here?

Reply:
The problem is that
Currently, when merging consecutive fake_quantize layers, we remain a fq generated by the 'outputs' scheme.
Since we remain 'input_node'(see https://github.com/openvinotoolkit/openvino/blob/master/tools/pot/openvino/tools/pot/graph/passes.py#L346)

e.g. relu - conv => [relu] - [fq outputs from relu] - [fq activations from conv] - [conv] => [relu] - [fq outputs from relu] - [conv]

So, we can't unify the q-bit for activation and weights of the convolution layer on the current implementation.
My idea is to give a priority value to fq for convolution in GNA3.5 then we can remain fq for conv activations not a outputs of previous layer
[relu] - [fq activations from conv] - [conv]

Reply:
Thanks, @bonhunko for the clarification. The problem you stated is more complicated in practice. I remember there were several cases that contradict to each other and prevent us from sticking to some specific strategy when merging output-input FQs. For now, let's keep what you did.